modules/social_features/social_like/social_like.tokens.inc

```diff
@@ -111,7 +112,7 @@ function social_like_tokens($type, $tokens, array $data, array $options, Bubblea
                   // We should only use the label of entities who have a label.
                   if ($ref_link_label = $commented_entity->label()) {
                     if ($commented_entity instanceof Node) {
-                      $commented_content_type = strtolower($commented_entity->type->entity->label());
+                      $commented_content_type = strtolower($commented_entity->getType());
```

Comment:
Isn't this the bundle instead of the label again? (Just making sure we dont miss this)

Reply:
Changed as per https://drupal.stackexchange.com/a/191485